swagger.json

```diff
@@ -4586,6 +4592,35 @@
                     }
                 }
         },
+        "TagMetadata": {
+            "type": "object",
+            "required": [
+                "id",
+                "name"
```

Comment:
This is a little awkward to require the tag name. Are we going to do anything with the tag name? I don't think this should update the existing tag's name. I'd be inclined to require only `id` for the input. In the future we may extend `external ids` to tags, which would allow them to specify `key:value` en lieu of `id`.